app/Services/RepositoryDetector.php

```
@@ -0,0 +1,124 @@
+        private int $teamId,
+    ) {}
+
+    public function detect(): RepositoryDetectionResult
+    {
+        $server = Server::query()
+            ->where('id', $this->serverId)
+            ->where('team_id', $this->teamId)
+            ->first();
+
+        if (! $server) {
+            return RepositoryDetectionResult::none();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Data\RepositoryDetectionResult' in method 'detect'. (undefined)

(StaticAccess)